build.psm1

```diff
@@ -1571,6 +1571,8 @@ function Start-PSxUnit {
 
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/debugging/README.md.https://aka.ms/vscclrdebugger should work
```powershell
Failed to complete request to "https://aka.ms/vscclrdebugger". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Reply:
Looks like the `release` branch no longer exists so that the URL is invalid.  This failure can be ignored for this PR.  Will get the aka.ms link fixed.